action.yml

```diff
@@ -30,6 +32,41 @@ inputs:
       list.
     required: false
     default: ""
+
+  test-results-pdf:
+    description: >-
+      Path to write test results report in PDF format
```

Comment:
Do we want to call out that this is not supported for macOS? In other words,

Path to write test results report in PDF format (currently not supported on macOS platforms).

In any case, the statement must be terminated by a period.